docs/core-concepts/workflow.md

```diff
@@ -12,7 +12,7 @@ can be tested can be modeled as a workflow in Kolena.
 Examples of workflows include:
 
 <div class="grid cards" markdown>
-- [:kolena-keypoint-detection-20: Keypoint Detection](https://github.com/kolenaIO/kolena/tree/trunk/examples/keypoint_detection) using images
+- [:kolena-widget-20: Example: Object Detection (2D) ↗](https://github.com/kolenaIO/kolena/tree/trunk/examples/object_detection_2d) using images
```

Comment:
Let's drop `Example: ` and `↗` from the name to be consistent.

Reply:
Will do